test/converters/converters.go

```diff
@@ -0,0 +1,21 @@
+package converters
```

Comment:
This file contains the interface that is required to be a package a test result converter. It must be possible to set files from outside(for example if someone wants to use a pre-filtered files list), need to return Junit4 xml test result, and needs to have a Detect method to see if the converter can run with the files included in the test result dictionary. (So a converter can run only if the dir has a TestSummaries.plist file for example)

Reply:
it would be a great package level comment

Reply:
AFAIK test folder is not locked at all by any Go convention.